andela-oaboluwarin/linksGraphQL
# initialize graphcool

#### What does this PR do?
- Initializes a basic backend server for the graphQL-based app.

#### Description of Task to be completed?
- Initialize graphQL
- Navigate to the graphcool playground to test (create mutations and then query created instances)

#### How should this be manually tested?
- Run `graphcool playground` in the terminal
- Copy the following into the editor pane:
```
mutation CreateGraphcoolLink {
  createLink(
    description: "The coolest GraphQL backend 😎",
    url: "https://graph.cool") {
    id
  }
}

mutation CreateRelayLink {
  createLink(
    description: "Highly performant GraphQL client from Facebook",
    url: "https://facebook.github.io/relay/") {
    id
  }
}
```
- Hit the `play` button and select each instance of the dropdown once each.
- overwrite data in editor with: 
```
{
  allLinks {
    id
    description
    url
  }
}

```
- You should see a return of the queried data.
